pkg/controller/keycloak/keycloak_reconciler_test.go

```diff
@@ -54,11 +54,11 @@ func TestKeycloakReconciler_Test_Creating_All(t *testing.T) {
 	//    7) Postgresql Service
 	//    8) Keycloak Service
 	//    9) Keycloak Discovery Service
-	//    10) Keycloak Probe ConfigMap
-	//    11) Keycloak StatefulSets
-	//    12) Keycloak Route
+	//    10) Keycloak Monitoring Service
+	//    11) Keycloak Probe ConfigMap
+	//    12) Keycloak StatefulSets
```

Comment:
Can we add the comment for the newly added 13th (14th ) item here?

Reply:
Done.